docs/getting-started/create-defend-policy-api.asciidoc

```diff
@@ -0,0 +1,819 @@
+[[create-defend-policy-api]]
+= Create an {elastic-defend} policy using API
+
+:frontmatter-description: Create and customize an {elastic-defend} policy through the API.
+:frontmatter-tags-products: [security, defend]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet-guide}/fleet-api-docs.html[{fleet} API]. You can repeat steps 2 and 3 to make more modifications to the {elastic-defend} policy.
+
+.Requirements
+[sidebar]
+--
+You must have the *{elastic-defend} Policy Management: All* <<endpoint-management-req,privilege>> to configure an integration policy.
+--
+
+[discrete]
+[[create-agent-policy]]
+== Step 1: Create an agent policy
+
+Make the following API call to create a new agent policy where you will add your {elastic-defend} integration. Replace `<KIBANA-VERSION>` with your version of {kib}.
+
+[source,console]
+----
+curl --user <username>:<password> --request POST \
+  --url 'https://<kibana-url>:5601/api/fleet/agent_policies' \
+  -H 'Accept: */*' \
+  -H 'Accept-Language: en-US,en;q=0.9' \
+  -H 'Connection: keep-alive' \
+  -H 'Content-Type: application/json' \
+  -H 'Sec-Fetch-Dest: empty' \
+  -H 'Sec-Fetch-Mode: cors' \
+  -H 'Sec-Fetch-Site: same-origin' \
+  -H 'kbn-version: <KIBANA-VERSION>' \ <1>
+  -d \
+'
+{
+  "name": "My Policy Name",
+  "description": "",
+  "namespace": "default",
+  "inactivity_timeout": 1209600
+}'
+----
+<1> `<KIBANA-VERSION>` to be replaced
+
+Make a note of the `<POLICY-ID>` you receive in the response. You will use this in step 2 to add {elastic-defend}.
+
+.Click to see example response
```

Comment:
Revise for more more neutral, less ableist language. Could also be something like "Click to expand example response" or just "Click for...":
```suggestion
.Click to display example response
```